File: backend/InsuranceAPI/Models/Address.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace InsuranceAPI.Models;

public partial class Address
{
    public long Id { get; set; }

    public string Street { get; set; } = null!;

    public string Number { get; set; } = null!;

    public int? Floor { get; set; }

    public int? Departament { get; set; }

    public string City { get; set; } = null!;

    public string Province { get; set; } = null!;

    public string Country { get; set; } = null!;

    [JsonIgnore]
    public virtual ICollection<Insured> Insureds { get; set; } = new List<Insured>();
}

File: frontend/src/app/models/Address.ts
export class Address{
    public street:string;
    public number:string;
    public city:string;
    public province:string;
    public country:string;
    public floor?:number;
    public departament?:number;
    public id?:number;   

    constructor(street:string,number:string,city:string,province:string,country:string,
                floor?:number,departament?:number,id?:number){
        this.street = street;
        this.number = number;
        this.city = city;
        this.province = province;
        this.country = country;
        this.floor = floor;
        this.departament = departament;
        this.id = id;
    }
}